firebase-firestore/src/test/java/com/google/firebase/firestore/remote/RemoteSerializerTest.java

```diff
@@ -118,16 +116,15 @@ public void setUp() {
   }
 
   private void assertRoundTrip(
-      FieldValue value, com.google.firestore.v1.Value proto, ValueTypeCase typeCase) {
-    com.google.firestore.v1.Value actual = value.getProto();
+      Value actual, com.google.firestore.v1.Value proto, ValueTypeCase typeCase) {
     assertEquals(typeCase, actual.getValueTypeCase());
     assertEquals(proto, actual);
-    assertTrue(ProtoValues.equals(value.getProto(), proto));
+    assertTrue(ProtoValues.equals(actual, proto));
   }
 
   @Test
   public void testEncodesNull() {
-    FieldValue value = wrap(null);
+    Value value = wrap(null);
     com.google.firestore.v1.Value proto = valueBuilder().setNullValueValue(0).build();
```

Comment:
Replace fully qualified `com.google.firestore.v1.Value` declarations in here with just `Value`?

Reply:
Done